src/test/java/io/vertx/test/core/LocalEventBusTest.java

```diff
@@ -1128,69 +1121,6 @@ public void testUnregisterThenUnsetEndHandler() {
     await();
   }
 
-  private void testUnregistrationOfUnregisteredConsumerCallsEndHandler(MessageConsumer<String> consumer, ReadStream<?> readStream) {
```

Comment:
Quite a few tests have been removed - are there corresponding tests for the new behaviour?